matchms/filtering/make_charge_scalar.py

```diff
@@ -1,15 +1,13 @@
+from deprecated.sphinx import deprecated
 from ..typing import SpectrumType
+from .make_charge_int import make_charge_int
 
 
+@deprecated(version='0.8.2', reason="Use expanded make_charge_int() instead.")
```

Comment:
Tried to make doc link with
```suggestion
@deprecated(version='0.8.2', reason="Use expanded :py:func:`.make_charge_int.make_charge_int()` instead.")
```
But this also shows up like that in ipython shell
```
<ipython-input-3-7dfb40347337>:1: DeprecationWarning: Call to deprecated function (or staticmethod) make_charge_scalar. (Use expanded :py:func:`make_charge_int()` instead.) -- Deprecated since version 0.8.2.
```
Which does not look so great as the thing is not importable like that by a library user. You decide whether this is you want to commit this suggestion.

Reply:
Thanks Stefan. I think I will skip this suggestions (too bad it doesn't render nicely).